tests/queries/0_stateless/02428_parameterized_view.sh

```diff
@@ -87,6 +88,9 @@ $CLICKHOUSE_CLIENT -q "SELECT * FROM test_02428_pv9(quantity=3)"
 $CLICKHOUSE_CLIENT -q "CREATE VIEW test_02428_pv10 AS SELECT Price FROM test_02428_Catalog WHERE Price={Pri:UInt64} ORDER BY Price"
 $CLICKHOUSE_CLIENT -q "SELECT * FROM test_02428_pv10(Pri=10)"
 
+$CLICKHOUSE_CLIENT -q "CREATE VIEW test_02428_pv11 AS SELECT * from ( SELECT Price FROM test_02428_Catalog WHERE Price={price:UInt64} )"
```

Comment:
let's also add a test for multiple nested subqueries, for example `CREATE VIEW test_02428_pv11 AS SELECT * from ( SELECT Price FROM test_02428_Catalog WHERE Price IN (SELECT number FROM numbers(10)))`

Reply:
I have added this to the test, with query parameter in nested subquery
`CREATE VIEW test_02428_pv12 AS SELECT * from ( SELECT Price FROM test_02428_Catalog WHERE Price IN (SELECT number FROM numbers({price:UInt64})) )`